libvast/vast/test/test_data.hpp

```diff
@@ -0,0 +1,16 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2022 Tenzir GmbH
+
+#pragma once
+
+#include <string>
+
+namespace vast::test {
+
+extern std::string EVE_JSON;
```

Comment:
Throughout this file there are a lot of uppercase variables, which totally goes against our naming conventions. There's also no documentation. Why is this?